subhub/sub/payments.py

```diff
@@ -78,6 +79,11 @@ def list_all_plans() -> FlaskListResponse:
     List all available plans for a user to purchase.
     :return:
     """
+    return _get_all_plans(), 200
+
+
+@cachetools.cached(cachetools.TTLCache(10, 600))
+def _get_all_plans():
```

Comment:
This private method is tested implicitly via the direct call in `list_all_plans`.  If you want the additional unit tests to validate the delegation, just call it out.